DataAccess/DbAccess/PostgresqlConfig.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace DataAccess.DbAccess
+{
+    public class PostgresqlConfig
+    {
+        public PostgresqlConfig(IConfiguration config)
```

Comment:
inject ça dans Configure dans un serviceCollectionExtensions que tu vas creer, voici un petit exemple
public static class ServiceCollectionExtensions
    {
        public static IServiceCollection UsePostgreSql(this IServiceCollection services,
            IConfigurationSection configuration)
            => services
                .Configure<PostgreSqlConfiguration>(configuration)